### PR TITLE
docs: clarify how to use flags in toml config file

### PR DESCRIPTION
**Pull request by ammgws**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ You must provide a config file that is passed using the `--config path_to_file.t
 - `icons` to assign icons to classes
 - `aliases` to assign alternative names to be displayed 
 - `general` to assign the separator and default icon
-- `options` to assign additional flags like `--no-names`
+- `options` to assign additional flags available in the cli interface. Replace hyphens from cli with underscores, e.g. `--no-names` would be `no_names` in the config file.
```
Comment (pedroscaff) [inline README.md:38]: I think it makes no sense that the `-` becomes `_` in the config file but I haven't changed it for so long that I didn't know about it anymore :smile:.

**Comment by pedroscaff** [comment]
thanks for the PR @ammgws

File: README.md (hunk 1 above)
Comment (pedroscaff) [reply README.md:38]: I wonder if people use it like that already, or if we could change the `toml` config to also use `-` without breaking configs. I assume you use it like that already @ammgws ?

Comment (ammgws) [reply README.md:38]: I only started using it the day I submitted the PR, so either is fine for me. Keeping it the same between cli and toml would be the most intuitive in my opinion.

Comment (pedroscaff) [reply README.md:38]: :+1: so I'll change it

**Pull request merged**